resources/views/pages/demo/index.blade.php

```diff
@@ -144,36 +147,111 @@
+                    <x-input.checkbox label="Checkbox checked" checked />
+                    <x-input.checkbox label="Checkbox disabled" checked disabled />
+                    <x-input.code label="Code" />
+                    <x-input.date label="Date" value="2024-12-31" :fullWidth="false" />
+                    <x-input.datetime-local label="Datetime local" value="2024-12-31 00:00:00" :fullWidth="false" />
+                    <x-input.email label="Email" value="test@example.com" :fullWidth="false" />
+                    <x-input.file label="File" :fullWidth="false" />
+                    <x-input.image label="Image" :fullWidth="false" />
+                </div>
+                <div>
+                    <x-input.number label="Number (stepper)" value="1234.5" :fullWidth="false" />
+                    <x-input.password value="password" :fullWidth="false" />
+                    <x-input.password-confirmed value="password" :fullWidth="false" />
+                    <x-input.search :fullWidth="false" />
+                    <x-input.select label="Select with preselected value" :options="[5 => 'Integer value 5', 4 => 'Integer value 4', '3' => 'String value 3', '2' => 'String value 2']" value="3" :fullWidth="false" />
+                    <x-input.select label="Select with only one option, not required" :options="['Only option']" :fullWidth="false" />
+                    <x-input.select label="Select with only one option, required" required :options="['Only option and required lorem ipsum']" :fullWidth="false" />
+                    <x-input.select label="Select with long options" required :options="['A long option label lorem ipsum', 'Another long option label lorem ipsum']" :fullWidth="false" />
+                </div>
+            </div>
+        </x-form>
     </x-section>
+
+    <x-form x-on:submit.prevent="alert('Preventing submission')" validate>
+        <x-section class="mb-8">
+            <x-section-header>
+                <x-slot name="title">
+                    <h4>Inline Form with only required fields</h4>
+                </x-slot>
+                <x-slot name="actions">
+                    <div>Valid: <span x-text="isValid"></span></div>
+                    <div>Sending: <span x-text="isSending"></span></div>
+                    <x-form-actions />
+                </x-slot>
+            </x-section-header>
+            <x-input.text name="text" requiredSilent inline help="Some helpful text describing this input" />
+            <x-input.user requiredSilent label="{{ __res('user', 1) }}" value="Scott" inline help="Scott preselected" />
+            <x-input.user requiredSilent inline help="Search user" />
+            <x-input.textarea label="{{ __res('message', 1) }}" requiredSilent inline maxlength="20" help="Some helpful text describing this input" />
+            <x-input.checkbox label="Checkbox" checked inline requiredSilent />
+            <x-form-actions inline />
+        </x-section>
+    </x-form>
```

Comment:
Usually `<x-section />` wrap forms but in this case it's the other way around to leverage alpine's scope.